types/npm-license-crawler/index.d.ts

```diff
@@ -1,16 +1,18 @@
-// Type definitions for npm-license-crawler 0.1
+// Type definitions for npm-license-crawler 0.1.1
```

Comment:
This won't work as DT only tracks major and minor versions of packages

Reply:
I thought it was the version of @types/npm-license-crawler. I've [reset to latest version.](https://github.com/mwittig/npm-license-crawler/releases/tag/V0.1.4)